google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/AppEngineDeploymentRunConfigurationEditor.java

```diff
@@ -102,7 +102,7 @@ public void actionPerformed(ActionEvent e) {
       }
     });
     userSpecifiedArtifactFileSelector.addBrowseFolderListener(
-        GctBundle.message("appengine.managedvm.config.user.specified.artifact.title"),
+        GctBundle.message("appengine.flex.config.user.specified.artifact.title"),
```

Comment:
Some of these classes have flex specific properties or do flex specific things, but conceivably we might unify that in teh future. If we don't we'll break out standard and flex configurations and rename.